corehq/apps/users/models.py

```diff
@@ -2732,6 +2732,45 @@ def get_role_name(self):
         else:
             return None
 
+    def _send_confirmation_email(self):
+        """
+        This sends the confirmation email to the invited_by user that their
+        invitation was accepted.
+        :return:
+        """
+        invited_user = self.email
+        subject = _('{} accepted your invitation to CommCare HQ').format(invited_user)
```

Comment:
What triggers this method? How is the translation language chosen in that context? Is it possible (assuming someone using a language different from the email owner's language was able to nefariously accept an invitation) that the owner of this email address might not be able to understand the message because it is not in their native language?

I'm also assuming that the primary reason we're sending this message it to alert the email owner that their address is being used in conjunction with CommCare activity, mostly to alert them of activity they are otherwise unaware of.

I'm not sure if there is anything we can do about that, but bringing it up so we can at least say we thought about it.

Reply:
it's usually triggered at the point where the user who accepts the invite. If it's a new user, then the language is English by default. but a good point. I think this might be a future to-do to make sure we switch translation contexts here